<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMainWindow</class>
 <widget class="QMainWindow" name="MyMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MyMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="add_file">
            <property name="text">
             <string>Add file(s)...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_directory">
            <property name="text">
             <string>Add directory...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remove_checked">
            <property name="text">
             <string>Remove checked</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delete_checked">
            <property name="text">
             <string>Delete checked</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="checkAll">
            <property name="text">
             <string>Check All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uncheckAll">
            <property name="text">
             <string>UnCheck All</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="tree_view">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="password_0">
        <property name="text">
         <string notr="true">Maximum lenght 32</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="password_label">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="password_1">
        <property name="text">
         <string notr="true">Maximum lenght 32</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Confirm:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="viewpass">
        <property name="text">
         <string>View Pass</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="generator">
        <property name="text">
         <string>Generator</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QLineEdit" name="encrypt_filename">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="encrypt_rename">
        <property name="text">
         <string>Incremental Rename encrypted item(s) to:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="delete_success">
        <property name="text">
         <string>Delete original item after encryption/decryption</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="encrypt">
        <property name="text">
         <string>Encrypt</string>
        </property>
        <property name="icon">
         <iconset resource="rsc.qrc">
          <normaloff>:/pixmaps/icon.png</normaloff>:/pixmaps/icon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="decrypt">
        <property name="text">
         <string>Decrypt</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_File"/>
    <addaction name="actionAdd_Directory"/>
    <addaction name="actionCheck_All"/>
    <addaction name="actionUncheck_All"/>
    <addaction name="actionRemove_checked"/>
    <addaction name="actionDelete_checked"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuList">
    <property name="title">
     <string>List</string>
    </property>
    <addaction name="actionLoadList"/>
    <addaction name="actionSave_item_list"/>
    <addaction name="actionRefresh_view"/>
    <addaction name="actionAuto_resize_columns"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
    </widget>
    <addaction name="actionConfiguration"/>
    <addaction name="actionDark_Theme"/>
    <addaction name="menuLanguage"/>
   </widget>
   <widget class="QMenu" name="menuPassword_Generator">
    <property name="title">
     <string>Utils</string>
    </property>
    <addaction name="actionPassword_Generator"/>
    <addaction name="actionHash_Calculator"/>
    <addaction name="actionArgon2_tests"/>
   </widget>
   <widget class="QMenu" name="menuInfos">
    <property name="title">
     <string>Infos</string>
    </property>
    <addaction name="actionAbout_Arsenic"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuList"/>
   <addaction name="menuConfig"/>
   <addaction name="menuPassword_Generator"/>
   <addaction name="menuInfos"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_File">
   <property name="text">
    <string>Add File(s)</string>
   </property>
  </action>
  <action name="actionAdd_Directory">
   <property name="text">
    <string>Add Directory</string>
   </property>
  </action>
  <action name="actionLoadList">
   <property name="text">
    <string>Load item list</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave_item_list">
   <property name="text">
    <string>Save item list</string>
   </property>
  </action>
  <action name="actionRefresh_view">
   <property name="text">
    <string>Refresh view</string>
   </property>
  </action>
  <action name="actionRemove_checked">
   <property name="text">
    <string>Remove checked</string>
   </property>
  </action>
  <action name="actionDelete_checked">
   <property name="text">
    <string>Delete checked</string>
   </property>
  </action>
  <action name="actionCheck_All">
   <property name="text">
    <string>Check All</string>
   </property>
  </action>
  <action name="actionUncheck_All">
   <property name="text">
    <string>Uncheck All</string>
   </property>
  </action>
  <action name="actionAuto_resize_columns">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto resize columns</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionPassword_Generator">
   <property name="text">
    <string>Password Generator</string>
   </property>
  </action>
  <action name="actionHash_Calculator">
   <property name="text">
    <string>Hash Calculator</string>
   </property>
  </action>
  <action name="actionAbout_Arsenic">
   <property name="text">
    <string>About Arsenic</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionArgon2_tests">
   <property name="text">
    <string>Argon2 tests</string>
   </property>
  </action>
  <action name="actionDark_Theme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark Theme</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionFrench">
   <property name="text">
    <string>French</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>encrypt</tabstop>
  <tabstop>decrypt</tabstop>
  <tabstop>password_0</tabstop>
  <tabstop>password_1</tabstop>
  <tabstop>encrypt_rename</tabstop>
  <tabstop>encrypt_filename</tabstop>
 </tabstops>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
